### ISSUE TITLE
genesis.bootstrap debootstrap fails if no qemu specified

### ISSUE THREAD

**Issue by aogier** [saltstack/salt#43387]
### Description of Issue/Question

Hi, I've just noticed that genesis debootstrap process fail if no static_qemu is passed on state.

Relevant traceback:

```
Traceback (most recent call last):
  File "/home/ubuntu/salt/salt/cli/caller.py", line 197, in call
    ret['return'] = func(*args, **kwargs)
  File "/home/ubuntu/salt/salt/modules/genesis.py", line 202, in bootstrap
    exclude_pkgs=exclude_pkgs,
  File "/home/ubuntu/salt/salt/modules/genesis.py", line 432, in _bootstrap_deb
    qemu=_cmd_quote(static_qemu), root=_cmd_quote(root)
  File "/usr/lib/python2.7/pipes.py", line 269, in quote
    for c in file:
TypeError: 'NoneType' object is not iterable
```

Since qemu does not seem to be necessary in order to finish chroot install, I'd like to fix it making `static_qemu` paramter actually optional, eg. skipping [executable copy](https://github.com/saltstack/salt/blob/fc587f784a15b0c3f3dcf24767b776c9f7f1c3a8/salt/modules/genesis.py#L430) in new env what do you think ?

Also, some test are needed (I've missed that the last genesis PR I've sent, sorry).

Thank you, regards

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)

```sh
salt-call --local genesis.bootstrap deb /tmp/cane arch=amd64 flavor=xenial repo_url=http://archive.ubuntu.com/ubuntu exclude_pkgs=vim,vim pkgs=bash -l debug
```

### Versions Report
```
Salt Version:
           Salt: 2016.11.6-174-gfc587f7
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: 1.7.3
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: 1.5.2
       M2Crypto: 0.21.1
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Nov 19 2016, 06:48:10)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: 0.6.8
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: Ubuntu 16.04 xenial
        machine: x86_64
        release: 4.4.0-92-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by garethgreenaway** [comment]
@aogier Thanks for the report and the PR!

**Comment by aogier** [comment]
fixed in 57cccd75d08a390ffb0a0722f872b782ef610452

thanks, ciao
